### ISSUE TITLE
Comparison Operators

### ISSUE THREAD

**Issue by NhanAZ** [Muqsit/arithmexp#9]
Something like:
```php
$parser = Parser::createDefault();

$expression = $parser->parse("2 + 3 > 5");
var_dump($expression->evaluate()); // bool(false)

$expression = $parser->parse("2 + 3 => 5");
var_dump($expression->evaluate()); // bool(true)
```
Comparison Operators: https://www.php.net/manual/en/language.operators.comparison.php

**Comment by Muqsit** [comment]
I have thought about implementing comparison operators (along with logical operators), but I am questioning whether they are a part of arithmetic expressions (mathematical calculators for example usually do not perform comparisons).

**Comment by NhanAZ** [comment]
I think it should be added. This feature is gradually being added on scientific calculators and is known as "Verify mode".

**Comment by Muqsit** [comment]
Implemented in [v1.0.0](https://github.com/Muqsit/arithmexp/releases/tag/1.0.0)
